docs/howto/mock_services.md

```diff
@@ -78,126 +78,169 @@ ZIO Test provides a framework for mocking your modules.
 
 ## Creating a mock service
 
-Assuming you are following the [module pattern][doc-use-module-pattern], for an `AccountListener` service we put the _capability tags_ and the access
-helpers (within `>` object) in the service companion object:
-
-```scala mdoc:invisible
-trait AccountEvent
-```
+We'll be assuming you've read [How to use modules and layers][doc-use-modules-and-layers] guide. In the main sources we define the _service_, a module alias and _capability accessors_. In test sources we're defining the _mock object_ which will hold _capability tags_ and mock layer machinery.
 
 ```scala mdoc:silent
 import zio.test.mock._
+import zio.test.mock.internal.MockRuntime
```

Comment:
Feels like this `MockRuntime` is pretty pervasive to be in an internal package. Also can we come up with an lias so we don't have to write `Has[MockRuntime]` everywhere? Maybe use the traditional pattern where type `MockRuntime = Has[MockRuntime.Service]`?

Reply:
Renamed to `Mock` and moved one package up. Instead old `Mock` became `MockFactory` and moved to internal. Regarding the alias - that was my initial approach, but then I had those weird cyclic dependency errors in 2.11, which seemed to be caused by the package object/alias to `MockRuntime`.

Reply:
Yeah, take a look at where we put the object and alias in ZIO Core and ZIO Test for the test implementations to see the best we came up with to work around this.